### ISSUE TITLE
[Feature Request] Registering context menu options

### ISSUE THREAD

**Issue by FallenStar08** [ShinyHobo/BG3-Modders-Multitool#210]
Simple suggestion to add an option to register two items to the windows context menu :

.pak files -> Unpack .pak files  
folder -> Pack mod

Without having to use the GUI. This would speed up testing etc. Something similar to [WitchyBND ](https://github.com/ividyon/WitchyBND)for EldenRing

**Comment by ShinyHobo** [comment]
You mean to the explorer context menu (ie add registry keys) or the toolbar at the top (new menu options)?

**Comment by FallenStar08** [comment]
Yep, explorer context menu, so that you could right click a pak file and unpack it from the explorer directly :)

**Comment by ShinyHobo** [comment]
I can look into adding an option to the configuration menu to add/remove this. This being a portable application means the registry value will be invalidated if someone moves their workspace.
